packages/wonder-blocks-icon/components/icon.test.js

```diff
@@ -0,0 +1,275 @@
+// @flow
+import * as React from "react";
+import * as Core from "@khanacademy/wonder-blocks-core";
+import icons from "../util/icon-assets.js";
+import {getPathForIcon, viewportPixelsForSize} from "../util/icon-util.js";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+// We mock things out so that we're in control of what really gets rendered.
+// Means we can test that we're using addStyle to generate the component
+// and then verify what that component has going on.
```

Comment:
I wonder if eventually we'll have a `wonder-blocks-test-tools` package. 🤔

Reply:
It seems likely.